src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -24,22 +24,39 @@ import "src/ray/protobuf/experimental/instance_manager.proto";
 // Following fields represents the Cluster Resources autoscaler interested
 // in.
 
-// Represents an anti-affinity constraint. A bundle with this constraint
-// can't be allocated to a node that has a label with the same name and
-// value. This is used to implement placement group anti-affinity.
+// Represents a placement constraint to specify how an associated resource
+// request should be placed with other entity, e.g.
+//  1. Other resource requests (e.g. gang scheduling)
+//  2. Nodes (e.g. specific instance type)
 //
-// For placement group, the label_name is "_PG" (reserved),
-// and the label_value is the placement group id.
-message AntiAffinityConstraint {
```

Comment:
I think autoscaler should generate a label map approprioately from the constraints, and we should decouple the constraint (which is mainly about resources requests) from the derived node labels. 

So I am keeping the pg id in the value rather then embed it in a key.